rascal-vscode-extension/src/extension.ts

```diff
@@ -101,7 +101,7 @@ function registerTerminalCommand(context: vscode.ExtensionContext, client:Langua
 
 function buildJVMPath(context: vscode.ExtensionContext) :string {
 	const jars = ['rascal-lsp.jar', 'rascal.jar', 'rascal-core.jar', 'typepal.jar'];
-	return jars.map(j => context.asAbsolutePath('./dist/' + j)).join(path.delimiter);
+	return jars.map(j => context.asAbsolutePath(path.join('.', 'dist', 'j'))).join(path.delimiter);
```

Comment:
I think you don't need `'.'`  in front, thats more a shell thing. if you just don't have a `/` in front of it, it will be a relative path anyway right? (I'm saying this since `./` is not a windows thing)